lib/google/ads/google_ads/config.rb

```diff
@@ -62,6 +64,15 @@ def initialize(&block)
         def configure(&block)
           yield self
         end
+
+        def http_proxy=(uri)
```

Comment:
Can you add a getter too in case people need to check which proxy they're using?